views/users/list.ejs

```diff
@@ -10,16 +14,29 @@
       <th>role</th>
       <th>unit</th>
       <th>department</th>
+        <th>validé par</th>
     </tr>
 
   <% users.forEach(function(user){ %>
-    <tr>
-      <td><%= user.id %></td>
-      <td><%= user.firstName %></td>
-      <td><%= user.lastName %></td>
-      <td><%= user.role %></td>
-      <td><%= user.unit %></td>
-      <td><%= user.department %></td>
-    </tr>
+            <tr>
+                <td><%= user.id %></td>
+                <td><%= user.firstName %></td>
+                <td><%= user.lastName %></td>
+                <td><%= user.role %></td>
+                <td><%= user.unit %></td>
+                <td><%= user.department %></td>
+
+                <% if (loggedInUser
+                    && loggedInUser.isValidator
+                    && !user.ValidatorId) { %>
```

Comment:
Je ne comprends pas la logique tu as choisi. Pourquoir avoir ajouté `&& !user.ValidatorId` ?

Reply:
Je n'ai rien ajouter depuis hier.
Si l'utilisateur de l'itération en cours n'a pas de validatorId, alors j'affiche le bouton pour le valider.